crates/aptos-batch-encryption/src/tests/typescript/runner.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+use anyhow::{anyhow, Context as _};
+use std::{
+    io::Write as _,
+    process::{Command, Stdio},
+    sync::Once,
+};
+
+static INIT: Once = Once::new();
+
+fn ts_init() {
```

Comment:
nit: if we want to make the PR smaller, we can omit these files and associated tests, and then add them in the second PR, where we also add the github workflow integrations? 🤔